lib/app/components/atoms/chat_bubble.dart

```diff
@@ -14,12 +16,15 @@ class ChatBubble extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BubbleSpecialThree(
-      text: text,
-      isSender: isSender,
-      color: colorChatBubbleBg, //TODO: 将来的にdark mode対応予定
-      tail: true,
-      textStyle: const TextStyle(color: colorChatBubbleText, fontSize: 16),
+    return cb.ChatBubble(
```

Comment:
```suggestion
    return ChatBubble(
```
クラス名変更に伴って、`cb.ChatBubble`→`ChatBubble`に変更お願いします。

Reply:
ご指摘の通り修正しました！